liquidprompt

```diff
@@ -826,12 +828,44 @@ _lp_sb() {
     [[ -n "$1" ]] && echo -nE " $1 "
 }
 
+
+# Adds as much $3 character (e.g. spaces) between $1 and $2,
+# so as to make the resulting string the same width than the current terminal.
+_lp_fill() {
```

Comment:
Love this.

Just spitballing here, would it be useful to have a function/version where you input three strings, and one of them is centered in the prompt?

```
a bunch of stuff here ----------------- central stuff here ----------------- right stuff here
```
I'm not sure what would be better, centered between the side texts, or centered in the terminal window.

Reply:
Definitely centered between the side texts.
I guess this may be useful for some hypothetical future theme.